osx/build

```diff
@@ -0,0 +1,23 @@
+#! /bin/bash -xe
+
+set -o pipefail
+
+brew install gtk+3
```

Comment:
The build script is very dependent on brew, as include paths are hardcoded into the build script for reliability.  Including brew commands here is not a problem since the software will not be installed again.  Brew will issue a warning: `Warning: goffice-0.10.27 already installed`